static/styles/zulip.css

```diff
@@ -1410,9 +1413,8 @@ td.pointer {
     }
 
     .messagebox-content {
-        box-shadow: inset 0 0 0 2px hsl(215, 47%, 50%),
-            -1px -1px 0 0 hsl(215, 47%, 50%), 1px 1px 0 0 hsl(215, 47%, 50%),
-            -1px 1px 0 0 hsl(215, 47%, 50%), 1px -1px 0 0 hsl(215, 47%, 50%);
+        outline: 1px solid hsla(217, 64%, 59%, 0.6);
+        border-radius: 4px;
```

Comment:
This is the core of this PR -- we now show a selected message with this thinner rounded outline instead of the old box-shadow.